src/auth/oauth2client.ts

```diff
@@ -901,28 +867,28 @@ export class OAuth2Client extends AuthClient {
    * are PEM encoded certificates.
    * @param callback Callback supplying the certificates
    */
-  getFederatedSignonCerts(): Promise<FederatedSignonCertsResponse>;
+  getFederatedSignonCerts(): Promise<Certificates>;
   getFederatedSignonCerts(callback: GetFederatedSignonCertsCallback): void;
   getFederatedSignonCerts(callback?: GetFederatedSignonCertsCallback):
-      Promise<FederatedSignonCertsResponse>|void {
+      Promise<Certificates>|void {
     if (callback) {
       this.getFederatedSignonCertsAsync()
-          .then(r => callback(null, r.certs, r.res))
+          .then(r => callback(null, r))
```

Comment:
Unrelated to your change, but `callback` will be called twice if it throws.

Reply:
Yeah, mistakes were made 😆 This is fixed over in #403